### PR TITLE
Update README.md and chart/README.md

### PR DESCRIPTION
**Pull request by PhanLe1010**
https://github.com/longhorn/upgrade-responder/issues/47

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,10 +12,12 @@ For example, Longhorn uses Upgrade Responder project to create the public metric
 
 ## How it works
 1. Upgrade Responder server is set up at an application-specified endpoint.
-1. A response JSON file (e.g. [upgrade-response.json](#response-json-config-example)) is provided to the server as the response to the incoming request.
+1. A response JSON file (e.g. [response-config.json](#response-config-example)) is provided to the server as the response to the incoming request.
    1. In Kubernetes deployment, the response JSON is normally configured as a Config Map and can be easily swapped out when a new version is available.
+1. A request schema JSON file (e.g. [request-schema.json](#request-schema-example)) is provided to the server to verify the format of incoming requests.
 1. The application now can make HTTP requests in a certain format to the application-specified endpoint periodically, and get the information about the latest available version.
    1. Server will expect each instance of the application makes a request **once per hour**.
+   2. The application can also send additional useful telemetric
```
Comment (c3y1huang) [inline README.md:20]:
```suggestion
   1. The application can also send additional useful telemetric.
```

File: README.md [hunk 2]
```diff
@@ -95,14 +100,17 @@ Upon success, you should see a dashboard similar to:
...
     "appVersion": "v0.8.1",
-    "extraInfo": {
-        "fieldKey": "value"
-    }
+    "extraTagInfo": {
+        "tagKey": "value"
+    },
+   "extraFieldInfo": {
+      "fieldKey": "value"
    }
```
Comment (c3y1huang) [inline README.md:111]:
```suggestion
    "extraFieldInfo": {
       "fieldKey": "value"
    }
```

**Review by c3y1huang — CHANGES_REQUESTED** [review]
Others LGTM.

File: README.md (hunk 1 above)
Comment (PhanLe1010) [reply README.md]: Thank you. Fixed

File: README.md (hunk 2 above)
Comment (PhanLe1010) [reply README.md]: Thank you. Fixed

**Review by c3y1huang — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -114,24 +122,48 @@ See our [example](./example) for how to use the client package.
...
-	"Versions": [{
-		"Name": "v1.0.0",
-		"ReleaseDate": "2020-05-30T10:20:00Z",
-		"Tags": ["latest"]
+	"versions": [{
+		"name": "v1.0.0",
+		"releaseDate": "2020-05-30T10:20:00Z",
+		"tags": ["latest"]
 	}]
 }
```
Comment (innobead) [inline README.md:133]: I feel I said this before about whether the current client still can decode this new response format correctly? IIRC, it should be YES, it seems the builtin go json decoder.

File: README.md [hunk 4]
```diff
@@ -95,14 +100,17 @@ Upon success, you should see a dashboard similar to:
...
 Your application should send one `POST` request every `requestIntervalInMinutes`. 
 The request's body should be in the format:
-   ```json
-   {
+```json
+{
     "appVersion": "v0.8.1",
-    "extraInfo": {
-        "fieldKey": "value"
+    "extraTagInfo": {
```
Comment (innobead) [inline README.md:106]: It will be good to explain what's difference between `extraTagInfo` and `extraFieldInfo`, so the user will know when to use which.

**Review by innobead — CHANGES_REQUESTED** [review]
In general, LGTM. However, some comments to resolve would help the doc better.

File: README.md (hunk 3 above)
Comment (PhanLe1010) [reply README.md:133]: Yeah, go json decoder will still understand the new format. I have tested with Longhorn v1.2.4

**Review by innobead — APPROVED** [review]

**Pull request merged**